README.md

```diff
@@ -28,56 +9,39 @@
 [![project group documentation](https://img.shields.io/badge/MDBook-View%20Documentation-blue)][gh-pages]
 
 
-**FIX ME**
-
 <!--
  Provide a short introduction about your project group. Make sure to include any
  relevant links to information about your group.
 -->
 
-Welcome to the repository for the {{GROUP_NAME}} Project Group! This is the
-repository we use to organise our work. Please refer to our [charter] as well
+Welcome to the repository for the Stable MIR Librarification Project Group! Our goal is to provide a stable
+API based on the rust compiler mid-level intermediate representation (MIR) that can be used as the foundation
+for development of tools that want to perform sophisticated analyses and make stronger guarantees about the
+behavior of Rust programs.
+
+
+This is the repository we use to organise our work. Please refer to our [charter] as well
 as our [github pages website][gh-pages] for more information on our goals and
 current scope.
 
 [charter]: ./CHARTER.md
-[gh-pages]: https://rust-lang.github.io/{{GROUP_SLUG}}
-
-**/FIX ME**
+[gh-pages]: https://rust-lang.github.io/project-stable-mir
```

Comment:
Do you happen to know what we need to do to get this link to start working? (Right now it 404's)

Reply:
I think we need to configure the github action. Let me fix that!

Reply:
I published #3 to fix the workflow and I have also configured our Page section inside project Settings to use the root of gh-pages.